### ISSUE TITLE
How can filter be implemented for the columns? I.e.; the column-specific search ?

### ISSUE THREAD

**Issue by FakhruddinNalawala** [ashvin27/react-datatable#53]
I want to use filters in the column, however, while the documentation mentions filters, it doesn't show how to actually use it...

**Comment by mkikets99** [comment]
@FakhruddinNalawala 
Well, i read the code and can give answer:

Seems like filter (or search) filters globally

@ashvin27 can I add functionality to search box with "keywords" like `key`:`search value`?
I think this would be good addition to search box when in config you can change search box to extended mode.

**Comment by ashvin27** [comment]
Actually, I'm thing to add column specific search in this user can add a search for each column.

**Comment by mkikets99** [comment]
@ashvin27 
you mean like this?
https://datatables.net/examples/api/multi_filter.html

**Comment by ashvin27** [comment]
@mkikets99 
Yes, correct. It would be in the footer or in the header anywhere the user wants.

**Comment by huntersinclairtiv** [comment]
Has column filters or a way to do an AND/OR filter been added yet?
